lerna.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
shouldn't this be in the /ts folder like in the sso ? 
Because is there any need to keep npm stuff in the root folder?